sdk/testutil/retry/retry.go

```diff
@@ -28,6 +28,10 @@ type Failer interface {
 
 	// FailNow is called when the retrying is abandoned.
 	FailNow()
+
```

Comment:
We probably don't need these changes which I recommended anymore since we cannot fully implement the testing.TB interface.